server/data.py

```diff
@@ -139,53 +139,27 @@ def _get_het_hom_index(self, data, index, use_gt, gp_thres, calc_info):
                 gt = s[0]
                 if gt == '1|1' or gt == '1/1':
                     hom_alt_i.append(i)
-                elif not (gt == '0|0' or gt == '0/0'):
+                elif (gt == '1|0' or gt == '1/0' or gt == '0|1' or gt == '0/1'):
```

Comment:
no need to start doing now but the data should really be encapsulated in classes and e.g. variant and genotypes would handle internally all these representations and logic here would just as is_het() etc.
proper variant parsin from vcfs can also be handled there in one place instead of parsin raw data where used. This get unmaintainable really fast as it is very error prone